web_redeploy.sh

```diff
@@ -1,7 +1,26 @@
 #! /bin/bash
+set -x
+
+ORIG_TEACHER_DASHBOARD_DIR=`grep -i "Alias /teacher-dashboard" ./docker-files/zeeguu-web/apache-zeeguu.conf | awk '{ print $3 }'`
+NEW_TEACHER_DASHBOARD_DIR="/opt/Zeeguu-Teacher-Dashboard-React/zeeguu-teacher-dashboard/build"
 
 docker restart zeeguu-web-dev
 
-#To consider this which is faster:
-#docker exec zeeguu-api-core-dev service apache2 reload
+#Start the python projects in develop mode
+docker exec zeeguu-web-dev python /opt/Zeeguu-Web/setup.py develop
```

Comment:
shouldn't starting the projects in dev mode should be the business of prepare for develop? 
if you run the setup develop it means that all the changes to the code will be visible directly at the next restart of the python interpreter. this is why `docker exec apache2ctl restart` should suffice for the web_redeploy... because it forces the reloading of the WSGI modules and thus the reloading of the python code from the now mapped sources.